cluster/build.gradle

```diff
@@ -203,3 +206,41 @@ shadowJar {
     )
   }
 }
+
+
+task generateSnappydataZeppelinInterpreterJar(type: Jar, dependsOn: compileScala) {
```

Comment:
This task should be integrated with the task which generates our distribution. I don't remember exactly. Please check with Amogh.